src/views/ipv/page/sorry-could-not-confirm-details.njk

```diff
@@ -14,6 +14,30 @@
   <p class="govuk-body">{{ 'pages.sorryCouldNotConfirmDetails.content.paragraph1' | translate }}</p>
   <p class="govuk-body">{{ 'pages.sorryCouldNotConfirmDetails.content.paragraph2' | translate }}</p>
 
+  {% if context == "deleteDetails" %}
```

Comment:
This screen now has two variants depending on whether it's reuse or repeat-fraud-check, so perhaps call this `deleteDetailsReuse`?